seed/static/seed/partials/analyses.html

```diff
@@ -0,0 +1,60 @@
+<div class="page_header_container">
+    <div class="page_header">
+        <div class="left page_action_container">
+            <a ui-sref="organizations" class="page_action"><i class="fa fa-chevron-left"></i> {$:: 'Organizations' | translate $}</a>
+        </div>
+        <div class="page_title">
+            <h1>{$:: org.name $}</h1>
+        </div>
+        <div class="right page_action_container">
+        </div>
+    </div>
+</div>
+<div class="section_nav_container">
+    <div class="section_nav" ng-include="::urls.static_url + 'seed/partials/accounts_nav.html'"></div>
+</div>
+<div class="section">
+    <div class="section_header_container">
+        <div class="section_header fixed_height_short">
+            <div class="left section_action_container">
+            	<h2><i class="fa fa-user"></i> {$:: 'Analyses' | translate $}</h2>
+           	</div>
+        </div>
+    </div>
+        <div class="section_content_container">
+        <div class="section_content">
+            <div class="table_list_container">
+                <table id="analyses-table" class="table table-striped table_highlight_first">
+                    <thead>
+                        <tr>
+                            <th translate>Analysis Name (User Defined)</th>
+                            <th translate>Actions</th>
+                            <th translate>Number of Properties</th>
+                            <th translate>Type</th>
+                            <th translate>Run Status</th>
+                            <th translate>Run Date</th>
+                            <th translate>Run Duration</th>
+                            <th translate>Run Author</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr ng-repeat="analysis in analyses  | filter:filter_params:strict">
+                            <td>{$:: analysis.name $}</td>
+                            <td>
+                            	<i class="glyphicon glyphicon-pencil" aria-hidden="true"></i>
+                            	<i class="glyphicon glyphicon-trash" aria-hidden="true"></i>
+                            </td>
+                            <td>{$:: analysis.number_of_analysis_property_views $}</td>
+                            <td>{$:: analysis.service $}</td>
+                            <td class="analysis-status {$:: analysis.status.toLowerCase() $}">{$:: analysis.status $}</td>
```

Comment:
No strong opinions from me here. I personally like this approach vs the ng-class, but I was never really a fan of the object you have to build for ng-class. @axelstudios thoughts?